libs/expo/betterangels/src/lib/screens/ClientHMISEdit/basicForms/PersonalInfo/PersonalInfoFormHmis.tsx

```diff
@@ -20,23 +21,27 @@ export function PersonalInfoFormHmis() {
   const {
     control,
     setValue,
+    watch,
     formState: { errors, isSubmitting },
   } = useFormContext<TPersonalInfoFormSchema>();
 
+  const dobQuality = watch('dobQuality');
+
+  const isDobDisabled =
+    dobQuality === 'DONT_KNOW' ||
+    dobQuality === 'NO_ANSWER' ||
+    dobQuality === 'NOT_COLLECTED' ||
+    dobQuality === '';
+
+  useEffect(() => {
+    if (isDobDisabled) {
+      setValue('birthDate', undefined);
```

Comment:
this isn't working as intended. The payload still contains the original birthdate. It only _appears_ to work for `NOT_COLLECTED` because clarity clears dob when dobquality is `NOT_COLLECTED`. 

`
[GraphQL req] updateHmisClientProfile {"data": {"additionalRaceEthnicityDetail": "", "birthDate": "2025-12-03", "californiaId": "", "dobQuality": "NO_ANSWER", "gender": ["NOT_COLLECTED"], "genderIdentityText": "", "id": "1", "livingSituation": null, "nameMiddle": "", "nameSuffix": "NO_ANSWER", "preferredLanguage": null, "raceEthnicity": ["NOT_COLLECTED"], "veteran": "NOT_COLLECTED"}}
`